client/components/SelectMenu/SelectMenu.tsx

```diff
@@ -61,7 +61,7 @@ const SelectMenu: React.FC<SelectMenuProps> & {
       value={value}
       name={name}
       onValueChange={onValueChange}
-      required={required}>
+      required={required ? true : false}>
```

Comment:
I would recommend to change this variable to `isRequired,` and using [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)
```suggestion
      required={required ?? false}>
```

Reply:
Done! Thanks for teaching me something new!